### ISSUE TITLE
Cannot save TIFF files with clipping info

### ISSUE THREAD

**Issue by dorphalsig** [python-pillow/Pillow#7008]
I am opening a TIFF file with clipping path data (see https://github.com/python-pillow/Pillow/issues/6025), I now want to save it with another name (nothing else was done to it)

```python
from PIL import Image
img = Image.open('images/path test.tif')
ex = ""
try:
    img.save("path test public.tif")
except Exception as e:
    ex=e
```

error message is "path test public.tif: Failed to allocate memory for custom tag binary object (-1578999712 elements of 1 bytes each)."

when I print the exception I see: RuntimeError('Error setting from dictionary')
file is attached inside a zip
[Path test.zip](https://github.com/python-pillow/Pillow/files/10950847/Path.test.zip)

**Comment by dorphalsig** [comment]
update tried it with another file that also has clipping info and got the same prob. updated tip file is attached
[Path test.zip](https://github.com/python-pillow/Pillow/files/10950851/Path.test.zip)

**Comment by radarhere** [comment]
Hmm. That doesn't quite match what I'm seeing, so let me ask - what happens if you delete the EXIF key from the TIFF tags? Does the script run successfully then?
```python
from PIL import Image
img = Image.open('images/path test.tif')
del img.tag_v2[34665]
ex = ""
try:
    img.save("path test public.tif")
except Exception as e:
    ex=e
```

**Comment by dorphalsig** [comment]
Hi. Nope. exact same error

**Comment by radarhere** [comment]
Ok, thanks. I've put together a potential fix, that should be part of Pillow 9.5.0 when it is released on April 1.

I'll check back in around then to confirm that it does indeed also resolve the problem in your environment.

**Comment by radarhere** [comment]
PR #7053 has been merged. Does that fix your problem? To test, you can either try and [install Pillow from source](https://pillow.readthedocs.io/en/stable/installation.html#building-from-source), or you can wait a day or so for Pillow 9.5.0 to be released.
